lib/ohai/plugins/init_package.rb

```diff
@@ -20,12 +20,6 @@
   provides "init_package"
 
   collect_data(:linux) do
-    package_name = nil
-
-    if File.exists?("/proc/1/comm")
-      package_name = File.open("/proc/1/comm").gets.chomp
-    end
-
-    init_package package_name
+    init_package File.exists?("/proc/1/comm") ? File.open("/proc/1/comm").gets.chomp : 'init'
```

Comment:
I'm not a huge fan of default values that you potentially change later on.  Without a comment it's often hard to understand why the initial value is set to something like init.  When you explicitly say if this file doesn't have content then use init it becomes a bit more clean to me that you chose init due to another value.